deps.ffmpeg/40-aom.ps1

```diff
@@ -47,6 +48,7 @@ function Configure {
         '-DENABLE_TOOLS:BOOL=OFF'
         '-DENABLE_NASM:BOOL=ON'
         "-DAOM_TARGET_CPU=$($TargetCPUs[$Target])"
+        $(if ( $Target -eq 'arm64' ) { '-DCONFIG_RUNTIME_CPU_DETECT=0' })
```

Comment:
Why is this added?

Reply:
Like with vpx I had issues with this when porting, but it compiles fine with current Visual Studio and version 3.9.1. Issues might occur only during runtime as well.